themes/frontierline/inc/customizer.php

```diff
@@ -94,6 +94,24 @@ function frontierline_customize_register($wp_customize) {
     'type'        => 'checkbox',
   ));
 
+  // Register featured image aka post thumbnail option and control
+  $wp_customize->add_setting('frontierline_post_thumbnail', array(
+    'capability'        => 'edit_theme_options',
+    'default'           => '1',
+    'sanitize_callback' => 'frontierline_sanitize_checkbox',
+    'type'              => 'theme_mod',
+  ));
+
+  $wp_customize->add_control('frontierline_post_thumbnail', array(
+    'label'       => esc_html__('Featured images', 'frontierline'),
+    'description' => esc_html__('Enable featured images for posts. (Posts without featured images will get a random fall back.)', 'frontierline'),
```

Comment:
This description may be slightly confusing. If you disable this, there will be actually no fallback image if I understand the code correctly.